libraries/core-react/src/Snackbar/Snackbar.tsx

```diff
@@ -34,25 +35,25 @@ const StyledSnackbar = styled.div.attrs(() => ({
   }
 `
 
-type Props = {
-  /* Controls the visibility of the snackbar */
+export type SnackbarProps = {
+  /**  Controls the visibility of the snackbar */
   open?: boolean
-  /* How long will the message be visible in milliseconds */
+  /** How long will the message be visible in milliseconds */
   autoHideDuration?: number
-  /* Callback fired when the snackbar is closed by auto hide duration timeout */
+  /** Callback fired when the snackbar is closed by auto hide duration timeout */
   onClose?: () => void
-  /* Media query from which the snackbar will be horizontal centered */
+  /** Media query from which the snackbar will be horizontal centered */
   leftAlignFrom?: string
 } & HTMLAttributes<HTMLDivElement>
 
-export const Snackbar = ({
+export const Snackbar: FC<SnackbarProps> = ({
```

Comment:
@wenche Why `FC` here? I thought we should use `forwardRef` and named functions instead

Reply:
See [commit message](https://github.com/equinor/design-system/pull/808/commits/0f002dd56cd81e0868f0a3e3c0bc2c8365c91a15). It didn't have forwardRef and adding it will be a breaking change.

Reply:
Oh.. So we should add `FC` to those who didn't have `forwardRef`? What breaking change will adding `forwardRef` make? I'm pretty sure I've added forwardRef's to some of my components so I should probably go back to do it your way.. 😬